vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh

```diff
@@ -183,6 +183,8 @@ installFIPS() {
         echo "installing linux-headers-fips..."
         apt_get_install 5 10 120 linux-headers-fips || exit $ERR_LINUX_HEADER_INSTALL_TIMEOUT
         ln -s /usr/src/linux-fips-headers-4.15.0-1039 /usr/src/linux-azure-headers-4.15.0-1002
+        mv -f /usr/src/linux-headers-4.15.0-1002-azure-fips /usr/src/linux-headers-4.15.0-1002-azure-fips1
```

Comment:
uname -r ?

Reply:
on 1039*

Reply:
Last Thursday, there seems to be some upstream repo changes which break the fips+gpu vhd build. @anujmaheshwari1 was nice to submit a PR (https://github.com/Azure/AgentBaker/pull/1474) to fix the build issue, however, there is runtime provisioning issue (/usr/sbin/dkms build -m nvidia -v 470.57.02 -k  4.15.0-1002-azure-fips always fails). This PR fixes the gpu driver build issue, but the cse still returns 84 during.  Since I have another PR to upgrade fips kernel version anyway (https://github.com/Azure/AgentBaker/pull/1497), and with this new kernel, fips or fips+gpu both works, I will not spend more time on this fips+gpu issue any more.